File: 2.Algorithm/08.Regular expression, String/숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    temp = ""
    answer = ""
    numbers = {"zero":"0",
        "one":"1",
        "two":"2",
        "three":"3",
        "four":"4",
        "five":"5",
        "six":"6",
        "seven":"7",
        "eight":"8",
        "nine":"9"}
    for i in s:
        try:
            answer += str(int(i))
        except:
            temp += i
            try:
                answer += numbers[temp]
                temp = ""
            except:
                pass
    return int(answer)

File: 2.Algorithm/08.Regular expression, String/숫자 문자열과 영단어/숫자 문자열과 영단어.java
class Solution {
    public int solution(String s) {
        String[] alphabets = {"zero","one","two","three","four","five","six","seven","eight","nine"};
        for(int i = 0; i < 10; i++){
            s = s.replaceAll(alphabets[i],""+i);
        }
        return Integer.parseInt(s);
    }
}